### PR TITLE
Front end Action cable: First step for real time chat.(All the cool kids review this pull request)

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

**Review by memunyawiri — APPROVED** [review]
:shipit:

File: app/assets/javascripts/channels/chat.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    // Called when the subscription is ready for use on the server
+  },
+
+  disconnected: function() {
+    // Called when the subscription has been terminated by the server
+  },
+
+  received: function(data) {
+    console.log(data)
+    console.log("Hello I am working")
```
Comment (dbugsy) [inline app/assets/javascripts/channels/chat.js]: is this meant to be here?

**Review by dbugsy — CHANGES_REQUESTED** [review]
As it is not test driven - how do you know you are building the right thing? How do you know it works? A capybara test to see a message being passed would help a lot.

File: app/assets/javascripts/channels/chat.js (hunk 1 above)
Comment (EOSullivanBerlin) [reply app/assets/javascripts/channels/chat.js]: No I will remove that and add the next pull request should have test coverage if we can figure it out.

File: Gemfile [hunk 2]
```diff
@@ -22,6 +22,8 @@ gem 'uglifier', '>= 1.3.0'
 
 gem 'devise'
 
+gem 'jquery-rails'
```
Comment (paulmillen) [inline Gemfile:25]: Have you looked at removing the jquery script from layout?  Possible conflicts with the gem?

**Review by petrakh — APPROVED** [review]
:shipit:

**Review by dbugsy — APPROVED** [review]

**Pull request merged**